src/lib/style/abstracts/_mixins.scss

```diff
@@ -64,6 +65,25 @@
   }
 }
 
+@mixin sticky-column() {
+  .stickyColumn {
+    right: 0;
+    position: sticky;
+    z-index: 1500;
```

Comment:
This will need to be tweaked:

https://user-images.githubusercontent.com/17580037/144623985-6a8e8031-75a8-4445-ab86-361c548ba0dd.mov

Reply:
Oops!  I'll figure out the right z-index.